### PR TITLE
Fix updating secrets topic

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/30436

### CODE REVIEW

**Comment by mburke5678** [comment]
@barleyer is this an issue you can review for QE? Thank you in advance.

**Comment by mburke5678** [comment]
Also fix in 3.11: https://docs.openshift.com/container-platform/3.11/dev_guide/secrets.html#secrets-and-the-pod-lifecycle

File: modules/builds-secrets-overview.adoc [hunk 1]
```diff
@@ -66,7 +66,7 @@ When you modify the value of a secret, the value used by an already running pod
 
 Updating a secret follows the same workflow as deploying a new container image. You can use the `kubectl rolling-update` command.
 
-The `resourceVersion` value in a secret is not specified when it is referenced. Therefore, if a secret is updated at the same time as pods are starting, then the version of the secret is used for the pod is not defined.
+The `resourceVersion` value in a secret is not specified when it is referenced. Therefore, if a secret is updated at the same time as pods are starting, then the version of the secret that is used for the pod is not defined.
```
Comment (codyhoag) [inline modules/builds-secrets-overview.adoc:69]: According to ISG, we should avoid if...then
>! if...then conj
Avoid using “then” to introduce an independent clause that follows an “if” clause. For example, write “If
you set a password, access is restricted,” not “If you set a password, then access is restricted.”

File: modules/nodes-pods-secrets-updating.adoc [hunk 2]
```diff
@@ -14,7 +14,7 @@ You can use the `kubectl rolling-update` command.
 
 The `resourceVersion` value in a secret is not specified when it is referenced.
 Therefore, if a secret is updated at the same time as pods are starting, then
-the version of the secret will be used for the pod will not be defined.
+the version of the secret that is used for the pod is not defined.
```
Comment (codyhoag) [inline modules/nodes-pods-secrets-updating.adoc:17]: Same comment on if...then

**Review by codyhoag — COMMENTED** [review]
A couple nits on `if...then` statements. They were already there, but figured I'd point them out. LGTM, otherwise

**Pull request merged**